ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -169,6 +177,10 @@ images:
   dockerfile_path: Dockerfile
   from: cuda-c9s-python-3.9
   to: cuda-rstudio-c9s-python-3.9
+- context_dir: amd/c9s-python-3.9
```

Comment:
Shall we build also the minimal version `amd-jupyter-minimal-c9s-python-3.9`?

Reply:
Yes that was built lines 153-160